crates/valence_protocol/src/packets/play/command_tree_s2c.rs

```diff
@@ -86,10 +86,10 @@ pub enum Parser<'a> {
     Dimension,
     GameMode,
     Time,
-    ResourceOrTag { registry: Ident<Cow<'a, str>> },
```

Comment:
is there a specific need for the switch from references and `Cow`s to owned strings?

Reply:
The rest of my code uses `String`s and they are easier to work with. Basically &str good for packets not for centrally used data structures.